system-tests/tests/smoke/cre/por_price_provider.go

```diff
@@ -209,13 +212,25 @@ func NewFakePriceProvider(testLogger zerolog.Logger, input *fake.Input, authKey
 		// Add more prices here as needed
 		pricesFloat64 := []float64{math.Round((rand.Float64()*199+1)*100) / 100, math.Round((rand.Float64()*199+1)*100) / 100}
 		pricesToServe[feedID] = pricesFloat64
+		testLogger.Info().Msgf("Generated raw float64 prices for feedID %s: %v", feedID, pricesFloat64)
```

Comment:
will that print meaningful data? or should we transform these floats first to strings with 2 precision?